plugins/com.google.cloud.tools.eclipse.appengine.deploy.ui/src/com/google/cloud/tools/eclipse/appengine/deploy/ui/DeployPropertyPage.java

```diff
@@ -202,7 +202,7 @@ private void evaluateFacetConfiguration() {
   private void updatePageBasedOnFacetConfig(boolean hasCorrectFacetConfiguration) {
     if (hasCorrectFacetConfiguration) {
       canSetMessage = true;
-      setErrorMessage(null);
+      content.getDataBindingContext().updateTargets();
```

Comment:
https://github.com/GoogleCloudPlatform/google-cloud-eclipse/issues/1564 filed to record the bug.